deploy-ghp.bat

```diff
@@ -0,0 +1,30 @@
+REM make sure git is clear:
+REM git status --porcelain
+
+git push origin --delete gh-pages
```

Comment:
The `gh-pages` approach has the drawback of increasing the repo size quite significantly. 

Also the GitHub Action to deploy to GitHub Pages will likely be default way soon.

Reply:
So not sure about recommending using `gh-pages` (even though it's been there for years)

Reply:
Is not "Github Pages" and "gh-pages" the same thing ?  
Any deployment must necessarily provide the static site somewhere, right ?
And the latest stage is enough - no need for history.  
If so, this is what the deploy script does.

Now deploying on github is not a reco in my mind, just a fast way to test, available to anybody.

Reply:
Even faster is local deploy - added in the section - this brings attention to the `dist` repo.